cookbooks/aws-parallelcluster-common/metadata.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+name 'aws-parallelcluster-common'
+maintainer 'Amazon Web Services'
+license 'Apache-2.0'
+description 'Installs AWS ParallelCluster'
```

Comment:
I'd say `description 'Common AWS ParallelCluster resources and attributes'`

Reply:
Done